app/backend/wells/tests/test_documents.py

```diff
@@ -13,13 +13,18 @@
 """
 import collections
 from django.test import TestCase
-from django.urls import reverse
+
 from rest_framework import status
 from rest_framework.test import APITestCase
+from rest_framework.reverse import reverse
 
+from gwells.settings import REST_FRAMEWORK
 from gwells.documents import MinioClient
 
 
+API_VERSION = REST_FRAMEWORK['DEFAULT_VERSION']
```

Comment:
I think we might have to test agains both v1 & v1 and not just the default/current version

Reply:
Since the v2 APIs are identical to v1 (outside of the upcoming minor change to /aquifers/<num>) I think we should only be testing the latest version, except in cases where the previous version API is different then the current one.

Otherwise we will have to double (then triple and so on) the API tests but for very little value of testing something uniquely different for each API.

Reply:
The postman tests might be a good place to test against both v1 & v2, I can help with that (probably tomorrow)

Reply:
We could verify that the output of a /v2/ API call is the same as a /v1/ call. That might be an easy way (so we don't duplicate tests) of verifying that the API hasn't changed in ways that are unexpected.